### ISSUE TITLE
Table valign middle bug

### ISSUE THREAD

**Issue by RekGRpth** [michaelrsweet/htmldoc#360]
1) test.html
```html
<table border="1" width="100%">
    <thead>
        <tr valign="middle">
            <th rowspan="2">Very very very long header</th>
            <th colspan="2">Long header</th>
            <th rowspan="2">Very long header</th>
            <th rowspan="2">Very very very long header</th>
            <th rowspan="2">Very very very long header</th>
            <th rowspan="2">Very very very long header</th>
            <th rowspan="2">Very very very long header</th>
            <th rowspan="2">Very very very long header</th>
            <th rowspan="2">Very very very long header</th>
        </tr>
        <tr>
            <th>Header</th>
            <th>Header</th>
        </tr>
    </thead>
</table>
```
2) command
```sh
htmldoc --charset utf-8 --header ... --footer ... --size a4 --outfile test.pdf test.html
```
3) test.pdf
[test.pdf](https://github.com/michaelrsweet/htmldoc/files/3558131/test.pdf)
4) test.png
![test](https://user-images.githubusercontent.com/3190488/63990563-dcb69280-cafd-11e9-92e2-144982f2f85c.png)

**Comment by michaelrsweet** [comment]
HTMLDOC does not support multi-row headers in THEAD like this... THEAD support is VERY limited (just to repeat a single row across multiple pages) as the table formatter follows HTML 3.2 rules...

I will see if there is anything obvious here, but do not expect a quick fix.

**Comment by michaelrsweet** [comment]
@RekGRpth What version of HTMLDOC are you using? I just tried this and got different (correct) results with Git master...

**Comment by RekGRpth** [comment]
Yes with default left margin there is no bag
```sh
htmldoc --webpage --left 1in --charset utf-8 --header ... --footer ... --size a4 --outfile test.pdf test.html
```
but with small left margin a bug appears
```sh
htmldoc --webpage --left 0.5in --charset utf-8 --header ... --footer ... --size a4 --outfile test.pdf test.html
```

**Comment by michaelrsweet** [comment]
@RekGRpth What version of HTMLDOC please?

**Comment by RekGRpth** [comment]
git master

**Comment by michaelrsweet** [comment]
OK, I was able to reproduce using the different margins. Investigating...

**Comment by michaelrsweet** [comment]
[master b79a27d] Fix a TABLE valign bug with rowspan (Issue #360)
